dashboard/app/helpers/levels_helper.rb

```diff
@@ -20,6 +20,22 @@ def url_from_path(path)
     "#{root_url.chomp('/')}#{path}"
   end
 
+  def create_channel(data = {}, src = nil)
+
+    result = ChannelsApi.call(request.env.merge(
+      'REQUEST_METHOD' => 'POST',
+      'PATH_INFO' => '/v3/channels',
+      'REQUEST_PATH' => '/v3/channels',
+      'QUERY_STRING' => src ? "src=#{src}" : '',
+      'CONTENT_TYPE' => 'application/json;charset=utf-8',
+      'rack.input' => StringIO.new(data.to_json)
+    ))
+    headers = result[1]
+
+    # Return the newly created channel ID.
+    headers['Location'].split('/').last
```

Comment:
Things seem to work now. I think I needed to restart my server after updating to this branch